YeomanTemplates/generator-choAdmin-boilerplate/generators/app/templates/package.json

```diff
@@ -0,0 +1,79 @@
+{
+  "name": "@canadahelps/<%= packageName %>",
+  "version": "<%= apiVersion %>",
+  "description": "<description>",
+  "main": "src/index<.ts|.js|.tsx>",
+  "scripts": {
+    "prepublishOnly": "npm test",
```

Comment:
@hdpinto Are we ok with these scripts, or does everyone seem to like the ones from `charity_registration` more?

Reply:
This one is for a shared lib, please remove
```suggestion
```